services/webhook/tests/test_voice_dispatch.py

```
@@ -0,0 +1,68 @@
+    def test_summary_markdown_sage_degraded(self) -> None:
+        from personas.code_reviewer.dispatch import _summary_markdown
+        from personas.code_reviewer.persona import CodeReviewEvaluation
+        
+        evaluation = CodeReviewEvaluation(
+            degraded_reason="test error",
+            findings=(),
+            dropped_hallucinations=0,
+            conclusion="neutral",
+        )
+        
+        title, summary = _summary_markdown(evaluation, voice="sage")
```

Ruff (0.15.20)

[warning] Unpacked variable `title` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)